cni/pkg/nodeagent/cni-watcher.go

```diff
@@ -200,3 +186,32 @@ func (s *CniPluginServer) ReconcileCNIAddEvent(ctx context.Context, addCmd CNIPl
 
 	return nil
 }
+
+func (s *CniPluginServer) getPodWithRetry(log *istiolog.Scope, name, namespace string) (*corev1.Pod, error) {
+	log.Debugf("Checking pod: %s in ns: %s is enabled for ambient", name, namespace)
+	maxStaleRetries := 10
```

Comment:
I know this wasn't the case in the previous code, but does it make sense to declare these as constants?